events/2023-10-in-person/index.qmd

```diff
@@ -80,4 +80,126 @@ some topics and less on others.
+    - Making sure everything is visible (can’t talk about things that are invisible)
+    - Clearer and more descriptive "events"
+    - Having time during meetings to dig into "user stories"/tasks
+        - Including adding some time estimates
+        - Every certain frequency, like every month
+        - Use them as the basis for a "focus" period
+    - Flow
+        - Keep in mind the overall roadmap
+        - Have a week of planning and deciding tasks
+        - Then have a period of working on those tasks over the next month
+    - [ ] Action: Add these potential items / workflows to our team docs
+- Misc
+    - We could include the timeline as part of our documentation
+
+### Update on progress, barriers and challenges
+
+- Getting lost down learning rabbit hole
+- Not having a clear path forward on what to work on
+- Feeling a bit managed vs led
+- Trust needs building up
+
+### Next steps to push progress forward
+
+- Moving forward, tracking progress, and building trust
+    - Building monthly sprints, with a period of roadmapping in those days
+    - Use Kanban board with constrained time periods to use for the focus phases, with dedicated time pre-phase to describe and plan together what needs to be done and how long each task takes
+    - Focus on smaller focused periods of working, with iterative development/feedback and smaller units of roadmapping.
+    - [ ] Action: Schedule time shortly 
+    - Have twice-weekly update meetings
+    - Limit external meeting
+    - Make use of Issue’s
+        - Issue templates for how to structure or fill in Issue/task
+        - [ ] Action: Richard make the first draft of template
+    - Make a PR template
+        - That includes a checklist of things to do
+        - Including “has unit test been added”
+    - Every update meeting comes with commit
+    - Nothing pushed to main, it has to go through PR and review
+    - If one of us are assigned or asked for review, work on it
+    - Always have someone assigned to everything
+    - Not working on side projects
+    - Annelli will attend an update meeting on
+- Tasks to do next
+    - Converting over the digital copies and updating documentation
+        - [ ] Action: Signe and Kris will convert
+    - Focusing on diagrams and visualizations in design docs, and updating the existing ones and adding more ones
+    - Have more definitions and lists on the definitions of terms
+    - Build up Actions/CI
+    - Pre-commits setup
+    - Basic infrastructure for working together
+    - Make issue template that includes a very detailed requirements for functionality (including input and output of functions/code)
+    - Review process?
+        - PR
+- Misc items
+    - Ask Mjolner SQL injection (and looking into things)
+    - Frictionless data, pull it in?
+    - Workflow, each project/package/app would have its own design doc and diagrams, etc?
+- Requirements for how to work together effectively as a team
```

Comment:
We also talked about best workflow practices such as aiming for multiple commits per day and pushing our work at the end of the day